package/wpe/wpeframework-interfaces/wpeframework-interfaces.mk

```diff
@@ -34,4 +34,8 @@ else ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_BUILD_TYPE_PRODUCTION),y)
         WPEFRAMEWORK_INTERFACES_CONF_OPTS += -DBUILD_TYPE=Production
 endif
 
+ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_CDM),y)
```

Comment:
Is this change related? If yes, then why it's needed now and not before?

Reply:
it was also done before